runtime/criusupport/criusupport.cpp

```diff
@@ -419,7 +490,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 
 		/* Run internal checkpoint hooks, after iterating heap objects */
 		if (FALSE == vmFuncs->runInternalJVMCheckpointHooks(currentThread)) {
-			currentExceptionClass = vm->criuSystemCheckpointExceptionClass;
+			currentExceptionClass = vm->criuJVMCheckpointExceptionClass;
```

Comment:
Question - why is not not in the VMCP?

Reply:
VMCP forces a classload which we want to avoid if the JVM is not started with `-XX:+enableCRIUSupport`

Reply:
That's actually not true - class loads are forced only for classes with fields/methods in the VMCP. You should be able to use `findKnownClass` here. This can be done in a future PR.